src/test/java/teammates/test/cases/datatransfer/AdminEmailAttributesTest.java

```diff
@@ -0,0 +1,140 @@
+                        FieldValidator.EMAIL_CONTENT_FIELD_NAME, FieldValidator.REASON_EMPTY,
+                        0) + Const.EOL
+                + getPopulatedErrorMessage(
+                        FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE, invalidAttributes.getSubject(),
+                        FieldValidator.EMAIL_SUBJECT_FIELD_NAME, FieldValidator.REASON_TOO_LONG,
+                        FieldValidator.EMAIL_SUBJECT_MAX_LENGTH);
+
+        assertFalse("Valid input", invalidAttributes.isValid());
+        assertEquals("Invalid input should return appropriate error string", expectedError,
+                StringHelper.toString(invalidAttributes.getInvalidityInfo()));
+    }
+
+    @Test
+    public void testGetValidityInfo() {
+        //already tested in testValidate() above
+    }
+
+    @Test
+    public void testSendDateForDisplay() {
+        Calendar calendar = formatDate(adminEmailAttributes.sendDate);
+        String dateForDisplay = TimeHelper.formatTime12H(calendar.getTime());
+        assertEquals(dateForDisplay, adminEmailAttributes.getSendDateForDisplay());
+    }
+
+    @Test
+    public void testCreateDateForDisplay() {
+        Calendar calendar = formatDate(adminEmailAttributes.createDate);
+        String dateForDisplay = TimeHelper.formatTime12H(calendar.getTime());
+        assertEquals(dateForDisplay, adminEmailAttributes.getCreateDateForDisplay());
+    }
+
+    /**
+     * Format Date for tests in this class.
+     */
+    public Calendar formatDate(Date date) {
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTime(date);
+        return TimeHelper.convertToUserTimeZone(calendar, Const.SystemParams.ADMIN_TIME_ZONE_DOUBLE);
+    }
+
+    @Test
+    public void testGetIdentificationString() {
+        assertEquals(new Date() + "/" + "subject of email", adminEmailAttributes.getIdentificationString());
+    }
+
+    @Test
+    public void testGetEntityTypeAsString() {
+        assertEquals("Admin Email", adminEmailAttributes.getEntityTypeAsString());
+    }
+
+    @Test
+    public void testGetBackupIdentifier() {
+        assertEquals("Admin Email", adminEmailAttributes.getBackupIdentifier());
+    }
+
+    @Test
+    public void testToEntity() {
+        AdminEmail adminEmail = adminEmailAttributes.toEntity();
+
+        assertEquals(adminEmailAttributes.subject, adminEmail.getSubject());
+        assertEquals(adminEmailAttributes.addressReceiver, adminEmail.getAddressReceiver());
+        assertEquals(adminEmailAttributes.groupReceiver, adminEmail.getGroupReceiver());
+        assertEquals(adminEmailAttributes.content, adminEmail.getContent());
+        assertEquals(adminEmailAttributes.sendDate, adminEmail.getSendDate());
+    }
+
+    @Test
+    public void testSanitizeForSaving() {
+        AdminEmailAttributes actualAdminEmail = adminEmailAttributes;
+        AdminEmailAttributes expectedAdminEmail = adminEmailAttributes;
```

Comment:
This will not work for two reasons:
- Java assigns Object variable by reference so essentially your `expected` and `actual` are always the same.
- Your attribute's subject and content are clean texts; sanitizing them will not cause any difference.

You should test this by simply creating a new attribute object, sanitize it, then check the subject and content fields post-sanitization.